packages/docs/src/pages/startup/browser-support.md

```diff
@@ -3,4 +3,16 @@ title: Browser support
 weight: 5
 ---
 
-The CMS design system is designed to support older and newer browsers through [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement). The current major version of the design system (2.0) follows the [2% rule](https://gds.blog.gov.uk/2012/01/25/support-for-browsers/): we officially support any browser above 2% usage as observed by [analytics.usa.gov](https://analytics.usa.gov/). Currently, this means that the newest versions of Chrome, Firefox, Safari, and Internet Explorer 11 and up are supported.
+The CMS design system follows the [2% rule](https://gds.blog.gov.uk/2012/01/25/support-for-browsers/): we officially support any browser above 2% usage as observed by [analytics.usa.gov](https://analytics.usa.gov/). **Currently, this means that the newest versions of Chrome, Firefox, Safari, Internet Explorer 11, Edge, and Samsung Internet are supported.**
+
+The CMSDS is designed to be compatible with different operating systems and browser versions in order to provide a broad range of support. Progressive enhancement and graceful degredation methodologies are used to accomplish this.
+
+## Progressive enhancement
```

Comment:
Do we need to provide all this information if the request was mainly about the list of browsers supported? This info doesn't feel very relevant to the page imo (i.e. graceful degradation)

Reply:
I'm good with that, I don't feel strongly about it. I was looking at the assets and styleguide sites and thinking about incorporating related info on the page. 

I'll update to remove it